src/test/java/nextstep/subway/path/application/PathServiceTest.java

```diff
@@ -64,11 +134,98 @@ void findShortestRoute() {
         when(lineRepository.findAll()).thenReturn(Arrays.asList(신분당선, 이호선, 삼호선));
 
         //when
-        PathResponse pathResponse = pathService.findShortestRoute(1L, 5L);
+        PathResponse pathResponse = pathService.findShortestRoute(1L, 5L, LoginMember.GUEST);
 
         //then
         assertThat(pathResponse.getStations().stream().map(station -> station.getName()).collect(Collectors.toList()))
                 .containsExactly("교대역","남부터미널역","양재역");
         assertThat(pathResponse.getDistance()).isEqualTo(5);
+        assertThat(pathResponse.getFare()).isEqualTo(1250);
+    }
+
+    @ParameterizedTest
+    @DisplayName("노선 추가요금 테스트")
+    @CsvSource(value = {"5:1350", "4:1550", "14:2250", "15:2550", "16:2650"}, delimiter = ':')
+    void lineExtraFeeTest(Long targetId, int expectedFare) {
+        //given
+        Long sourceId = 1L;
+        when(stationService.findStationById(sourceId)).thenReturn(stationMap.get(sourceId));
+        when(stationService.findStationById(targetId)).thenReturn(stationMap.get(targetId));
+        when(lineRepository.findAll()).thenReturn(Arrays.asList(일호선, 이호선, 삼호선, 신분당선, 경강선));
+
+        //when
+        int fare = pathService.findShortestRoute(sourceId, targetId, LoginMember.GUEST).getFare();
+
+        //then
+        assertThat(fare).isEqualTo(expectedFare);
+    }
+
+    @ParameterizedTest
+    @DisplayName("비회원 요금 테스트")
+    @CsvSource(value = {"8:1250", "9:1350", "10:1550", "11:2150", "12:2250", "13:2750"}, delimiter = ':')
+    void noLoginFeeTest(Long targetId, int expectedFare) {
+        //given
+        Long sourceId = 7L;
+        when(stationService.findStationById(sourceId)).thenReturn(stationMap.get(sourceId));
+        when(stationService.findStationById(targetId)).thenReturn(stationMap.get(targetId));
+        when(lineRepository.findAll()).thenReturn(Arrays.asList(일호선, 이호선, 삼호선, 신분당선, 경강선));
+
+        //when
+        int fare = pathService.findShortestRoute(sourceId, targetId, LoginMember.GUEST).getFare();
+
+        //then
+        assertThat(fare).isEqualTo(expectedFare);
+    }
+
+    @ParameterizedTest
+    @DisplayName("비회원 요금 테스트")
```

Comment:
```suggestion
    @DisplayName("어른 요금 테스트")
```

---

전체적으로 테스트 구성 잘 해주셨고, 전체 테스트 정확성도 검증했습니다 👍 
고생 많으셨어요 😊

Reply:
감사합니다 :)